src/components/list/list.scss

```diff
@@ -16,63 +17,161 @@ $item-padding-top: 0px !default;
+
+  .md-subheader {
+    font-weight: 500;
+    font-size: 0.875em;
+  }
 }
 
 md-item {
-
+  &.md-no-style,
+  .md-no-style {
+    padding: $list-item-padding-vertical $list-item-padding-horizontal;
+    flex: 1;
+  }
+  &.md-with-secondary {
+    padding-right: $list-item-padding-horizontal;
+  }
+  &.md-clickable:hover {
+    cursor: pointer;
+  }
 }
 
-md-item-content {
+md-item, md-item > .md-no-style {
   display: flex;
+  justify-content: flex-start;
   align-items: center;
-  flex-direction: row;
-  box-sizing: border-box;
 
-  position: relative;
+  // Layout for controls in primary or secondary divs, or auto-infered first child
+  & > div.md-primary > md-icon,
+  & > div.md-secondary > md-icon,
+  & > md-icon:first-child, 
+  > md-icon.md-secondary {
+    width: $list-item-primary-icon-width;
+    margin-top: 12px;
+    margin-bottom: 12px;
+    box-sizing: content-box;
+    &:focus {
+      padding: 6px;
+      margin-right: -6px;
+      margin-top: 6px;
+      margin-bottom: 6px;
+    }
+  }
+  & > div.md-primary > md-checkbox,
+  & > div.md-secondary > md-checkbox,
+  & > md-checkbox:first-child,
+  md-checkbox.md-secondary {
+    position: relative;
+    top: -2px;
+    margin-top: 14px;
+    margin-bottom: 12px;
+    .md-label { display: none; }
+    &:focus {
+      padding: 6px;
+      margin-right: -6px;
+      margin-top: 8px;
+      margin-bottom: 6px;
+    }
+  }
 
-  padding: $item-padding-top $item-padding-right $item-padding-bottom $item-padding-left;
-}
+  & > md-icon:first-child {
+    margin-right: $list-item-primary-width - $list-item-primary-icon-width;
+  }
+  & > md-checkbox:first-child {
+    width: 3 * $baseline-grid;
+    margin-left: 3px;
+    margin-right: 29px;
+  }
+  & > .md-avatar:first-child {
+    width: $list-item-primary-avatar-width;
+    height: $list-item-primary-avatar-width;
+    margin-top: $baseline-grid;
+    margin-bottom: $baseline-grid;
+    margin-right: $list-item-primary-width - $list-item-primary-avatar-width;
+    border-radius: 50%;
+    box-sizing: content-box;
+  }
+
+  md-checkbox.md-secondary {
+    margin-right: 0px;
+  }
+  md-switch.md-secondary {
+    margin: 0;
+    position: relative;
+    right: -9px;
+  }
+  .md-secondary {
+    margin-left: $list-item-secondary-left-margin;
+  }
 
-/**
- * The left tile for a list item.
- */
-.md-tile-left {
-  min-width: 56px;
-  margin-right: -16px;
+  & > p, & > button.md-no-style > p {
+    flex: 1;
+    margin: 0;
+  }
 }
 
-/**
- * The center content tile for a list item.
- */
-.md-tile-content {
-  flex: 1;
 
-  padding: $baseline-grid * 2;
+md-item.md-2-line,
+md-item.md-2-line > .md-no-style,
+md-item.md-3-line,
+md-item.md-3-line > .md-no-style {
+  align-items: flex-start;
 
-  text-overflow: ellipsis;
+  .md-item-text {
+    flex: 1;
+    padding: $baseline-grid * 2 0;
+    text-overflow: ellipsis;
 
-  h3 {
-    margin: $list-h3-margin;
-    font-weight: $list-h3-font-weight;
-    font-size: $list-h3-font-size;
+    h3 {
```

Comment:
We should revisit these after PR #1985 lands, in case we can consolidate styles.